File: c1/parte1.c
#include <stdio.h>

/*L\tikz[remember picture]{\coordinate (icf);}L*/int maximo(int a, int b) {/*L\tikz[remember picture]{\coordinate (fcf);}L*/
  if (a > b) {
    return a;
  } else {
    return b;
  }
}

int potencia(int a, int b) {
  int i, prod = 1;
  for (i = 0; i < b; i++) {
    prod = a * prod;
  }
  return prod;
}

int main() {
  int a, b, maior, pot;
  printf("Entre com a e b\n");
  scanf("%d %d", &a, &b);
  maior = maximo(a, b);
  pot = potencia(a, b);
  printf("Maior: %d\n", maior);
  printf("a^b: %d\n", pot);
  return 0;
}

int maximo(int a, int b);

File: c1/parte1.py
'''L\tikz[remember picture]{\coordinate (ipf);}L'''def maximo(a, b):'''L\tikz[remember picture]{\coordinate (fpf);}L'''
    if a > b:
        return a
    else:
        return b

def potencia(a, b):
    prod = 1
    for i in range(b):
        prod = a * prod
    return prod

print("Entre com a e b")
a = int(input())
b = int(input())
maior = maximo(a, b)
pot = potencia(a, b)
print("Maior:", maior)
print("a^b:", pot)
